app/assets/stylesheets/course/course_users.scss

```diff
@@ -0,0 +1,8 @@
+.course-course-users {
+  .profile-picture {
+    .image > img {
+      margin-top: 1em;
+      max-height: $course-course-user-profile-picture;
```

Comment:
I put it as max-height so that it won't look oversized for certain screen configurations. Didn't fix down the height so that we can leave more space for the user's details on smaller screens instead of the photo taking up the space.

Would it be better to fix down the pic size and let the picture be above instead of beside the the user's details on smaller screens? Any more appropriate way?